pandas/io/sql.py

```diff
@@ -516,7 +516,8 @@ def read_sql(sql, con, index_col=None, coerce_float=True, params=None,
 
 
 def to_sql(frame, name, con, flavor='sqlite', schema=None, if_exists='fail',
-           index=True, index_label=None, chunksize=None, dtype=None):
+           index=True, index_label=None, chunksize=None, dtype=None,
+           indexes=None):
```

Comment:
Can this be included in v0.18.1 ? or will it be for v0.19.0.

I will also update the corresponding whatsnew/.txt file.